src/react-components/ui-root.js

```diff
@@ -440,7 +440,7 @@ class UIRoot extends Component {
     if (!this.props.forcedVREntryType || !this.props.forcedVREntryType.endsWith("_now")) {
       this.goToEntryStep(ENTRY_STEPS.audio_setup);
     } else {
-      setTimeout(this.onAudioReadyButton, 3000); // Need to wait otherwise input doesn't work :/
+      this.onAudioReadyButton(); // Need to wait otherwise input doesn't work :/
```

Comment:
Nice that we were able to get rid of this timeout (how?) but we should also remove the comment

Reply:
haha whoops. `2d_now` no longer really works as it used to -- it doesn't just enter the scene now, it requires one-click to enter. this was a small change to make it less painful, but a separate PR will be necessary to fully restore the feature. (it regressed when I did the overhaul to the landing page)